jwst/datamodels/fits_support.py

```diff
@@ -462,7 +462,7 @@ def _load_from_schema(hdulist, schema, tree, context, skip_fits_update=False):
         if not any(isinstance(hdu, fits.BinTableHDU) for hdu in hdulist if hdu.name != 'ASDF'):
             log.debug('Skipping FITS updating completely.')
             return known_keywords, known_datas
-        log.debug('Skipping FITS keyword updating. FITS Table and data HDU updating will occur.')
+        log.debug('Skipping FITS keyword updating. FITS BinTableHDU and ImageHDU updating will occur.')
```

Comment:
The debug text should be something like `"Skipping FITS keyword updating except for BinTableHDU and it's associated header keywords"`.  `ImageHDU` objects are never updated under any circumstances, because they are not stored separately in the `AsdfInFITS` tree.